cmd/controller/app/controller.go

```diff
@@ -276,7 +275,6 @@ func startLeaderElection(ctx context.Context, opts *options.ControllerOptions, l
 			OnStartedLeading: run,
 			OnStoppedLeading: func() {
 				log.Info("leader election lost")
```

Comment:
I think we might need to adapt this so the controller still exits when it loses an election 🙄 

/hold